io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
```

Comment:
is `missioncenter.io`

for now it looks like it is parked.

Reply:
Yes, it is owned by me, the web-site is not up yet